src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+# Revolutionaries Rework
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| SlamBamActionman<br>ScarKy0 | :x: No | TBD |
+
+## Overview
+
+Revolutionaries as a gamemode is a mechanically and thematically interesting concept; sedition against the station's established authority sounds fun and there is a lot of design space to work within to represent this in mechanics. Previous iterations of Revolutionaries (Revs) have however been seen as encouraging too much combat with too snowball-y rounds for either side, frustrating "metagameable" mechanics and design that both directly and indirectly encourages removing a sizeable number of players from the round. 
+
+This rework intends to address many of the critiques of the original Revs gamemode while still keeping to the theme of exaggerated "Space Communism", allowing for the Revs faction to take over the station via stealth, subterfuge, blatant opposition or violence. 
+
+This design document is inspired by ideas presented by the the following works and authors:<br>
+https://github.com/space-wizards/docs/pull/251 - CaelleakNavinski<br>
+https://github.com/space-wizards/docs/pull/146 - illdoodle<br>
+ScarKy0
+
+### Breakdown
+<sup>In this document, the default side players belong to will be referred to as either "crew" or "Command".</sup>
+
+Revolutionaries are a team conversion antag: a small number of players begin as antagonists, but are able to convert crew to antags, with the antag faction's strength largely being dependent on the number of conversion. As a conversion antag, a round with Revolutionaries should attempt to follow the stages below. 
+
+- Initial conversions
+  - This is the stage where the antag begin conversion crew - this should be done stealthily, and failing to do so may result in a swift win for crew. To their benefit the antag is given some kind of initial advantage, providing a quick power bump to get things going.
```

Comment:
This is a constant problem with snowballing antags in general. The design of any Revs mode that involves snowballing is going to have to counterbalance that a fluked round (i.e. "Revs are discovered immediately through misfortune") with recovery.

The critical flaw of current Revs is the "discovery" mechanic. The only thing that prevents the HOS from creating a mindshield line in front of Sec every shift is admin rules; any Revs refactor must find a way of resolving this.